src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx

```diff
@@ -51,20 +54,33 @@ class SettingsLayout extends React.Component {
   }
 }
 
+const StyledBreadcrumbWrapper = styled('div')`
+  display: flex;
+`;
+
+const SettingsHamburgerIcon = styled(Button)`
+  margin-right: 16px;
+  @media (min-width: ${p => p.theme.breakpoints[0]}) {
+    display: ${p => (p.isVisible ? 'block' : 'none')};
+  }
+`;
 const MaxWidthContainer = styled('div')`
   display: flex;
   max-width: ${p => p.theme.settings.containerWidth};
   flex: 1;
 `;
-
 const SidebarWrapper = styled('div')`
   flex-shrink: 0;
   width: ${p => p.theme.settings.sidebarWidth};
   background: #fff;
   border-right: 1px solid ${p => p.theme.borderLight};
   padding: ${space(4)};
+  @media (max-width: ${p => p.theme.breakpoints[0]}) {
+    display: ${p => (p.isVisible ? 'none' : 'block')};
+    position: sticky;
+    z-index: 1;
```

Comment:
Can we have it float over the content?

Reply:
Would be nice to see some animations in here too, shouldn't be hard to get this to slide in if we do something like animate it from `left: -100%` to `left: 0`.